Utilities/ITKv5Preparation/ReplaceOutdatedMacroNames.sh

```diff
@@ -30,7 +30,18 @@ echo "WARNING:  This script is not intended to be bullet-proof."
 echo "WARNING:  Please carefully review all changes made to ensure proper behavior."
 }
 
-ReplaceCXXString ITK_NOEXCEPT_OR_THROW ITK_NOEXCEPT
-ReplaceCXXString ITK_HAS_CXX11_STATIC_ASSERT ITK_COMPILER_CXX_STATIC_ASSERT
-ReplaceCXXString ITK_DELETE_FUNCTION ITK_DELETED_FUNCTION
-ReplaceCXXString ITK_HAS_CPP11_ALIGNAS ITK_COMPILER_CXX_ALIGNAS
+ReplaceCXXString ITK_NOEXCEPT_OR_THROW             ITK_NOEXCEPT
```

Comment:
Should these contents be in a folder named `Utilities/ITKv6Preparation/ReplaceOutdatedMacroNames.sh` (note the **v6**)?